infrastructure/template.yaml

```diff
@@ -94,6 +94,8 @@ Resources:
     Type: AWS::SQS::Queue
     DeletionPolicy: Delete
     UpdateReplacePolicy: Delete
+    Properties:
+      MessageRetentionPeriod: 1209600
```

Comment:
That's 14 days right?

So is that: It gives us 14 days to respond to an error and read off the data to fix things up?
Sounds like a reasonable window given we'll probably want to act much sooner if anything ever crops up here.

Reply:
Yeah exactly, see my commit message on [4ff34fb](https://github.com/alphagov/di-account-management-backend/pull/26/commits/4ff34fb9c7cb19dd9bc7066b00141d5c97bf0416) for my thinking there.